sdk/storage/azure-storage-blob/out/azure/storage/blob/_blob_client.pyi

```diff
@@ -0,0 +1,59 @@
+    def download_blob(self, offset: Optional[int]=..., length: Optional[int]=..., **kwargs: Any) -> StorageStreamDownloader: ...
+    def delete_blob(self, delete_snapshots: bool=..., **kwargs: Any) -> None: ...
+    def undelete_blob(self, **kwargs: Any) -> None: ...
+    def get_blob_properties(self, **kwargs: Any) -> BlobProperties: ...
+    def set_http_headers(self, content_settings: Optional[ContentSettings]=..., **kwargs: Any) -> None: ...
+    def set_blob_metadata(self, metadata: Optional[Dict[str, str]]=..., **kwargs: Any) -> Dict[str, Union[str, datetime]]: ...
+    def create_page_blob(self, size: int, content_settings: Optional[ContentSettings]=..., metadata: Optional[Dict[str, str]]=..., premium_page_blob_tier: Optional[Union[str, PremiumPageBlobTier]]=..., **kwargs: Any) -> Dict[str, Union[str, datetime]]: ...
+    def create_append_blob(self, content_settings: Optional[ContentSettings]=..., metadata: Optional[Dict[str, str]]=..., **kwargs: Any) -> Dict[str, Union[str, datetime]]: ...
+    def create_snapshot(self, metadata: Optional[Dict[str, str]]=..., **kwargs: Any) -> Dict[str, Union[str, datetime]]: ...
+    def start_copy_from_url(self, source_url: str, metadata: Optional[Dict[str, str]]=..., incremental_copy: bool=..., **kwargs: Any) -> Dict[str, Union[str, datetime]]: ...
+    def abort_copy(self, copy_id: Union[str, Dict[str, Any], BlobProperties], **kwargs: Any) -> None: ...
+    def acquire_lease(self, lease_duration: int=..., lease_id: Optional[str]=..., **kwargs: Any) -> BlobLeaseClient: ...
+    def set_standard_blob_tier(self, standard_blob_tier: Union[str, StandardBlobTier], **kwargs: Any) -> None: ...
+    def stage_block(self, block_id: str, data: Union[Iterable[AnyStr], IO[AnyStr]], length: Optional[int]=..., **kwargs: Any) -> None: ...
+    def stage_block_from_url(self, block_id: str, source_url: str, source_offset: Optional[int]=..., source_length: Optional[int]=..., source_content_md5: Optional[Union[bytes, bytearray]]=..., **kwargs: Any) -> None: ...
+    def get_block_list(self, block_list_type: Optional[str]=..., **kwargs: Any) -> Tuple[List[BlobBlock], List[BlobBlock]]: ...
+    def commit_block_list(self, block_list: List[BlobBlock], content_settings: Optional[ContentSettings]=..., metadata: Optional[Dict[str, str]]=..., **kwargs: Any) -> Dict[str, Union[str, datetime]]: ...
+    def set_premium_page_blob_tier(self, premium_page_blob_tier: Union[str, PremiumPageBlobTier], **kwargs: Any) -> None: ...
+    def get_page_ranges(self, offset: Optional[int]=..., length: Optional[int]=..., previous_snapshot_diff: Optional[Union[str, Dict[str, Any]]]=..., **kwargs: Any) -> Tuple[List[Dict[str, int]], List[Dict[str, int]]]: ...
+    def get_managed_disk_page_range_diff(self, offset: Optional[int]=..., length: Optional[int]=..., prev_snapshot_url: Optional[str]=..., **kwargs: Any) -> Tuple[List[Dict[str, int]], List[Dict[str, int]]]: ...
```

Comment:
We would like to remove this API completely - as the feature can still be accessed via the existing `get_page_ranges` API.

Reply:
We haven't reached an agreement to merge it across all SDKs, probably leave the managed_disk_diff API here for now